.github/workflows/check-change-and-comment.yml

```diff
@@ -0,0 +1,109 @@
+# **what?**
+# Checks that a file has been added/modified under the specified directory
+# And comment on the PR if the requested change is missing.
+#
+
+# Example Usage including required permissions
+
+# on:
+#   pull_request_target:
+#     types: [opened, reopened, labeled, unlabeled, synchronize]
+#
+# permissions:
+#   contents: read
+#   pull-requests: write
+#
+# name: Check Changelog Entry
+# jobs:
+#   changelog:
+#     uses: dbt-labs/actions/.github/workflows/changelog-check.yml@main
+#    with:
+#      changelog_comment: 'Thank you for your pull request! We could not find a changelog entry for this change. For details on how to document a change, see [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-changelog-entry).'
+#      path: '.changes/unreleased/**.yaml'
+#    secrets: inherit # this is only acceptable because we own the action we're calling
```

Comment:
Once this is an action this will need to be updated.

Reply:
I am not sure I follow here